### ISSUE TITLE
BenchHelper in numpytets issues deprication warning on travis

### ISSUE THREAD

**Issue by tblum** [bh107/bohrium#26]
```
Traceback (most recent call last):
  File "bohrium/test/python/numpytest.py", line 373, in <module>
    (res1,cmd1) = getattr(cls_inst,mth)(np_arys)
  File "/home/travis/build/bh107/bohrium/test/python/test_ndstencil.py", line 18, in test_ndstencil_1D
    return self.run(pseudo_arrays)
  File "/home/travis/build/bh107/bohrium/test/python/numpytest.py", line 283, in run
    res[k] = npzs[k]
  File "/usr/lib/python2.7/dist-packages/numpy/lib/npyio.py", line 231, in __getitem__
    return format.read_array(value)
  File "/usr/lib/python2.7/dist-packages/numpy/lib/format.py", line 440, in read_array
    shape, fortran_order, dtype = read_array_header_1_0(fp)
  File "/usr/lib/python2.7/dist-packages/numpy/lib/format.py", line 336, in read_array_header_1_0
    d = safe_eval(header)
  File "/usr/lib/python2.7/dist-packages/numpy/lib/utils.py", line 1132, in safe_eval
    import compiler
  File "/usr/lib/python2.7/compiler/__init__.py", line 27, in <module>
    DeprecationWarning, stacklevel=2)
DeprecationWarning: The compiler package is deprecated and removed in Python 3.x.
```
The strange thing is that travis is NOT running python3 by default.
Anyways made a quickfix: https://github.com/bh107/bohrium/commit/3e8c77776b39f610ca63525035b24162921396d6

We might want a more robust solution?

**Comment by safl** [comment]
This is just the thing for Mads :)

**Comment by madsbk** [comment]
This is a BUG in NumPy, they use the compiler module, which is a standard module in python2.7 but is removed in python3.
Your fix in 3e8c77776b39f610ca63525035b24162921396d6 is fine!
